### ISSUE TITLE
Screen Reader: Settings-About: QR code is not focused and announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74884]
## Action Performed
Prerequisite:
User is logged in

1. Using JAWS + Chrome, open the page https://new.expensify.com/settings/about
2. Navigate using TAB to 'App download links', press enter to activate
3. On the side panel pop up menu, navigate using TAB or G key to attempt to reach the QR code image
4. Observe the behavior

Other occurrences:
1. On Settings - Security - Two-factor authentication
2. On Settings - Profile - Share code

## Expected Result
The informative QR code image will be focused and announced to users by the screen reader.

## Actual Result
When users navigate through the 'App download links' side menu, the 'Expensify QR Code' image is neither focused or announced. During linear the QR code image is skipped from the screen reader users. Hence The QR code and its associated info will not will be available for screen reader users.

<svg viewBox="0 0 172 172" width="172" height="172" data-component="QRCode" data-element="Svg" data-source-file="index.js"><defs data-element="Defs" data-source-file="index.js"><linearGradient x1="0%" y1="0%" x2="100%" y2="100%" id="grad" data-element="LinearGradient" data-source-file="index.js"><stop offset="0" stop-color="rgb(255,0,0)" stop-opacity="1" data-element="Stop" data-source-file="index.js"></stop><stop offset="1" stop-color="rgb(0,255,255)" stop-opacity="1" data-element="Stop" data-source-file="index.js"></stop></linearGradient></defs><g data-element="G" data-source-file="index.js"><rect x="0" y="0" width="172" height="172" fill="#FCFBF9" data-element="Rect" data-source-file="index.js"></rect></g><g data-element="G" data-source-file="index.js">

## Area issue was found
Settings-About

## Failed Checkpoint
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User Impact
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested Resolution
-Give the QR code role="img" and provide a descriptive name to it by using an aria-label or use a <title> attribute to provide accessible name to the image that matches the logo name (Expensify QR code) and associate it with aria-labelled by attribute.

Refer:
https://www.magentaa11y.com/#/web-criteria/component/informative-image?tab=2

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1917" height="998" alt="Image" src="https://github.com/user-attachments/assets/8f2af179-f950-41f8-bc2a-1f1655dc9fc7" />

https://github.com/user-attachments/assets/c05f7aa6-ece1-4fdd-ae48-4c6f31688483

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6995315
**Issue reported by:** NA
**Slack conversation:** NA
